lib/actions/api.js

```diff
@@ -204,6 +211,15 @@ export function updateOtpUrlParams(state, searchId) {
       .forEach((key) => {
         params[key] = urlParams[key]
       })
+
+    // Populate the URL parameters for the mode buttons if those are specified in the currentQuery
+    if (currentQuery.modeButtons) {
+      params.modeButtons = currentQuery.modeButtons
```

Comment:
This was actually brought up a while ago, I think. The problem with this is that `modes` are different from `modeButtons`, since `modeButtons` refers to the selected buttons, each of which contains various modes in it that get activated or deactivated with the button. 

I've had to re-add support for directly specifying the modes in the currentQuery object for call taker and field trip.